tests/framework/results/test_observation.py

```diff
@@ -72,19 +78,20 @@ def test_stratified_observation__aggregate(
         stratifications=stratifications, filtered_pop=filtered_pop
     )
     aggregates = stratified_observation._aggregate(
-        pop_groups=groups,
+        pop_groups=groups,  # type: ignore [arg-type]
         aggregator_sources=aggregator_sources,
         aggregator=aggregator,
     )
     if aggregator == len:
         if stratifications:
-            stratification_idx = (
+            stratification_idx: set[tuple[str, ...] | str] = (
```

Comment:
How did you end up sorting this out?

Reply:
This is essentially in line typing this for both places.